source/dnode/mgmt/node_mgmt/src/dmEnv.c

```diff
@@ -51,26 +51,14 @@ static int32_t dmInitMonitor() {
 
 static bool dmCheckDiskSpace() {
   osUpdate();
-  if (!osDataSpaceAvailable()) {
-    dError("free disk size: %f GB, too little, require %f GB at least at least , quit",
-           (double)tsDataSpace.size.avail / 1024.0 / 1024.0 / 1024.0,
-           (double)tsDataSpace.reserved / 1024.0 / 1024.0 / 1024.0);
-    terrno = TSDB_CODE_NO_AVAIL_DISK;
-    return false;
+  if (!osDataSpaceSufficient()) {
```

Comment:
Still allow taosd to start on condition of disk shortage. At the same time, print some error messages. If possible, it is better to trigger real-time alarm at appropriate intervals relevant to disk usage of space. To allow OAM team to take action to prevent from hitting the condition of disk full.